### DISCUSSION TITLE
Begin a lineserie at a specific point on X axis

### DISCUSSION THREAD

**Discussion by wasilus** [Live-Charts/LiveCharts2#338] [Q&A] [answered]
Suppose I have 2 line series, one begins at 0 on X axis, the other one I want it to begin at a specific point on X axis. 
I wonder if that is possible and how, documentation? Thank you.

**Comment by beto-rodriguez** [comment, marked as answer]
Out of the box, you can use the `ObservablePoint` class to specify both.

https://github.com/beto-rodriguez/LiveCharts2/blob/docs/samples/ViewModelsSamples/Lines/XY/ViewModel.cs

**Reply by wasilus** [reply]
Thank you very much.
